fapi-client/src/test/scala/com/gu/facia/api/models/ImportanceTest.scala

```diff
@@ -0,0 +1,32 @@
+package com.gu.facia.api.models
+
+import com.gu.facia.client.models.CollectionConfigJson
+import org.scalatest.{Matchers, FlatSpec}
+
+class ImportanceTest extends FlatSpec with Matchers {
+
+  def collectionConfigJsonWithImportance(importance: Option[String]) =
```

Comment:
Is this indirection really adding anything? It'd be simpler to just use withDefaults directly, I think?